src/components/SlideButton/SlideButton.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <div class="wrapper">
+    <div class="slider-wrapper">
+      <div
```

Comment:
(nit) please consolidate this element onto a single-line (because it only has two attributes and the line wouldn't be very long) 

Also: please always separate "fixed"/static classes from dynamic-bindings (like I've written below)

```js
<div ref="slider" class="slider-track" :class="sliderClass">
```